utils/retry/retryexecutor.go

```diff
@@ -0,0 +1,58 @@
+package retry
```

Comment:
I think we can remove the *retry* package, because it includes only retryexecutor.go and retryexecutor_test.go

Reply:
Removed